src/core/model/index.ts

```diff
@@ -117,6 +117,23 @@ export class EthqlBlock implements EthqlBlock {
   }
 }
 
+export interface EthqlOmmerBlock
```

Comment:
@akhila-raju – maybe you could elaborate on why it was necessary to introduce an separate class for ommer blocks, and to overwrite only the miner property? Shouldn't this go in #102?

Reply:
Sure - the reasons why I introduced a separate class for ommers is because ommers don't contain transactions:

- transactions shouldn't be a property on ommers if they don't exist
- transactionsCount will always return null for ommers

Rather than carry down a check for whether the block is an ommer block, I decided to create a new class to simplify things.

The miner property was already EthqlAccount, it wasn't overwritten - just copied and pasted for EthqlOmmerBlock :)